edx/edx-app-ios
# iOS - improvement of the player progress bar slider

### Description

[LEARNER-7905](https://openedx.atlassian.net/browse/LEARNER-7905)

In this PR I improved the difficulty in moving the slider of the progress bar of the video. 

### How to test this PR
- Open any video.
- Try to slide the video progress slider, it should be more smooth as the previous version.
